pages/learn/more/masterclass.md

```diff
@@ -10,8 +10,9 @@ Feedback on the masterclasses are welcome and should be given by creating an iss
 
 Currently, the following masterclasses are available:
 
-* [{{ $names.cli }} masterclass]({{ $links.githubMain }}/balena-cli-masterclass) - a guide to getting started with the {{ $names.cli }}.
-* [{{ $names.cli }} advanced masterclass]({{ $links.githubMain }}/balena-cli-advanced-masterclass) - a guide to the advanced features available in {{ $names.cli }}.
+* [{{ $names.company.upper }} CLI masterclass]({{ $links.githubMain }}/balena-cli-masterclass) - a guide to getting started with the {{ $names.cli }}.
```

Comment:
Why the change here?

Reply:
Because it's lowercase and there is no .upper and .lower - though I should probably just add them.  I'll update this.

Reply:
Makes sense then, thanks for clarifying!

Reply:
I've pushed an update and updated all existing references